docs/resources/elb_logtanks.md

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Dedicated Load Balance (Dedicated ELB)"
+---
+
+# huaweicloud_elb_logtank
+
+Manage a ELB logtank resource within HuaweiCloud.
+
+## Example Usage
+
+### create with system template
+
+```hcl
+resource "huaweicloud_elb_logtank" "test" {
+  loadbalancer_id = var.loadbalancer_id
+  log_group_id    = var.group_id
+  log_topic_id    = var.topic_id
+}
+```
+
+### create with custom template
+
+```hcl
+variable "loadbalancer_id" {}
+variable "group_id" {}
+variable "topic_id" {}
+resource "huaweicloud_elb_logtank" "test" {
```

Comment:
this config is same with `create with system template`.

Reply:
done